### PR TITLE
fix: Update record schema

### PR DESCRIPTION
**Pull request by sbellone**
To reflect changes made in extraction.

### CODE REVIEW

**Review by bodinsamuel — APPROVED** [review]

File: docs/schema.md [hunk 1]
```diff
@@ -114,3 +117,15 @@ For this URL: <https://www.algolia.com/products/crawler/>
 ## Record splitting
 
 For better relevance, we can split records into multiple ones. We create all indices with the index settings `{ distinct: true, attributeForDistinct: 'url' }` to deduplicate them at search time.
+
+## JSON-LD
+
+We support a limited set of [JSON-LD](https://json-ld.org/) attributes. We expect the JSON-LD structure to follow the https://schema.org/ structure.
```
Comment (bodinsamuel) [inline docs/schema.md:123]:
```suggestion
We support a limited set of [JSON-LD](https://json-ld.org/) attributes. We expect the JSON-LD structure to follow the <https://schema.org/> structure.
```

File: docs/schema.md [hunk 2]
```diff
@@ -54,22 +54,25 @@ All properties that aren't marked as optional are present in the final record. O
   image?: string;
 
   /**
-   * The author of the page.
+   * The authors of the page.
    * - meta[property="article:author"]
+   * - `author` field of JSON-LD Article object: https://schema.org/Article
```
Comment (bodinsamuel) [inline docs/schema.md:59]: Add them before the selector to make it the order of priority clearer

File: docs/schema.md (hunk 1 above)
Comment (sbellone) [reply docs/schema.md]: Added, but it looks the same with or without those.

**Review by bodinsamuel — APPROVED** [review]

File: docs/schema.md (hunk 1 above)
Comment (bodinsamuel) [reply docs/schema.md]: it's because github is kind to autoparse url probably, but it's more correct like this

**Pull request merged**